internal/utils/strings.go

```diff
@@ -3,8 +3,24 @@ package utils
 import (
 	"math/rand"
 	"time"
+	"unicode"
 )
 
+// IsStringAlphaNumeric returns false if any rune in the string is not alpha-numeric.
+func IsStringAlphaNumeric(input string) bool {
+	if len(input) == 0 {
```

Comment:
I think you can safely remove that if block.

Reply:
Adjusted.